email_template_qweb/models/mail_template.py

```diff
@@ -33,7 +33,7 @@ def generate_email(self, res_ids, fields=None):
                     })
                     # Some wizards, like when sending a sales order, need this
                     # fix to display accents correctly
-                    if not isinstance(body_html, unicode):
+                    if not isinstance(body_html, str):
```

Comment:
I think we'll be better off with using https://github.com/OCA/OCB/blob/11.0/odoo/loglevels.py#L43 here